src/data/init/init-moves.ts

```diff
@@ -3259,7 +3259,7 @@ export function initMoves() {
       .attr(TeraBlastTypeAttr)
       .attr(TeraBlastPowerAttr)
       .attr(StatStageChangeAttr, [Stat.ATK, Stat.SPATK], -1, true, {
-        condition: (user, _target, _move) => user.isTerastallized() && user.isOfType(ElementalType.STELLAR),
+        condition: (user) => user.isTerastallized && user.isOfType(ElementalType.STELLAR),
```

Comment:
I'm not sure it's best to remove the unused params in cases like these since in the future if it's necessary to use them it's not obvious to developers that they're available to be used.

Reply:
Feel free to restore them then if you don't feel confident about that change:

```suggestion
        condition: (user, _target, _move) => user.isTerastallized && user.isOfType(ElementalType.STELLAR),
```